addon/components/mdc-tab-bar.js

```diff
@@ -110,7 +123,12 @@ export default Ember.Component.extend(MDCComponent, {
   },
   setTabActiveAtIndex(index, isActive) {
     if (get(this, 'links')) {
-      this.tabAt(index)._invoke({ stopPropagation() {}, preventDefault() {} }); // TODO: Probably shouldn't be calling private APIs or stubbing events
+      if (this.tabAt(index) && isActive) {
+        this.tabAt(index)._invoke({ stopPropagation() {}, preventDefault() {} }); // TODO: Probably shouldn't be calling private APIs or stubbing events
```

Comment:
Another TODO

Reply:
This one was added by you I think?

Reply:
Yep I didn't notice that it was moved, not added.